.github/workflows/build-and-test.yml

```diff
@@ -34,6 +34,35 @@ jobs:
     - name: Build all classes
       run: ./gradlew --stacktrace classes
 
+    - name: Install Node
+      uses: actions/setup-node@v4
```

Comment:
If Node is just installed for PNPM to run, we could consider instead enabling [standalone mode](https://github.com/pnpm/action-setup?tab=readme-ov-file#standalone) below.

Reply:
We have some types for Node installed, so I'm hesitant to drop it at this point.